.github/workflows/release-cycle.yml

```diff
@@ -142,6 +142,11 @@ jobs:
         run: bash scripts/release/workflow/pack.sh
         env:
           PRERELEASE: ${{ needs.state.outputs.is_prerelease }}
+      - name: Upload tarball artifact
+        uses: actions/upload-artifact@v3
```

Comment:
```suggestion
        uses: actions/upload-artifact@e9ef242655d12993efdcda9058dee2db83a2cb9b
```

Reply:
I'll accept the suggestion and add a comment.
There'd be already 2 technical debt items here:

1. To remove the comment for the [Changesets Action `body` parameter PR](https://github.com/changesets/action/pull/250) when they merge it
2. A comment to use `actions/upload-artifact@v3` when it's released.

Does this sound acceptable to you?

Reply:
It's not ideal but it's okay with me. We may get an automatic PR to update the action from Renovatebot.